contracts/polytone-tester/Cargo.toml

```diff
@@ -17,9 +17,8 @@ library = []
 
 [dependencies]
 cosmwasm-schema = { workspace = true }
-cosmwasm-std = { workspace = true }
+cosmwasm-std    = { version = "1.2.4", features = ["ibc3"] }
```

Comment:
why not use workspace true here? think this is the same as what's in the root

Reply:
fixed, left over from the multiple tries